src/pages/TermsAndConditions.js

```diff
@@ -0,0 +1,654 @@
+import React, { Component } from 'react'
+import TermsAndConditionsStyle from './TermsAndConditions.module.scss'
```

Comment:
Can shorten this to Style for consistency with other components.
You'll need to replace all references.